libs/ui/src/components/DisabledTab/DisabledTab.scss

```diff
@@ -4,7 +4,6 @@
   &--disabled {
     // overide o-tabs-item
     cursor: auto !important;
-    pointer-events: all !important;
```

Comment:
this was put here in order for tooltip to show

Reply:
yeah I figured, adding this makes disabled tab it clickable though

Reply:
any way to have pointer-events: none and still show tooltip? @daiagi @roiLeo

Reply:
from the top of my head i think you will need to make the tooltip trigger parent of the tab header

Reply:
yeah @click.prevent didnt work, tried other ways as well, not able to make it work, maybe we can close this issue and someone else can take over this one